src/backend/mailbox_manager/admin.py

```diff
@@ -135,6 +135,39 @@ def fetch_domain_expected_config_from_dimail(modeladmin, request, queryset):  #
         )
 
 
+@admin.action(description=_("Send pending mailboxes to dimail"))
+def send_pending_mailboxes(modeladmin, request, queryset):  # pylint: disable=unused-argument
+    """Send pending mailboxes"""
+    client = DimailAPIClient()
+
+    excluded_domains = []
+    for domain in queryset:
+        # do not check disabled domains
+        if domain.status != enums.MailDomainStatusChoices.ENABLED:
+            excluded_domains.append(domain.name)
+            continue
+
+        response = client.enable_pending_mailboxes(domain)
+        if response:
```

Comment:
I think this is always "True" because response is `{"failed_mailboxes": failed_mailboxes}`

I suggest to name "response" => "results" and therefore:
```suggestion
        if failed_mailboxes := results["failed_mailboxes"]:
```